app/models/site_config.rb

```diff
@@ -29,6 +29,7 @@ class SiteConfig < RailsSettings::Base
   field :community_description, type: :string, default: "A constructive and inclusive social network. Open source and radically transparent."
   field :community_member_description, type: :string, default: "amazing humans who code."
   field :community_member_label, type: :string, default: "user"
+  field :community_action, type: :string, default: "coding"
```

Comment:
Leaving this as a default for now, all defaults will be removed for future communities in a subsequent PR based on [this issue](https://github.com/thepracticaldev/dev.to/projects/15#card-37093809)